src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -228,6 +235,13 @@ private LayoutBlock buildChannelMentionBlock() {
         return blocks;
     }
 
+    private List<LayoutBlock> buildUserIdBlocks(UserDTO userDTO) {
+        List<LayoutBlock> blocks = new ArrayList<>();
+        blocks.add(SectionBlock.builder().text(PlainTextObject.builder().text("User ID: " + userDTO.getId()).build()).blockId(USER_ID.getId()).build());
```

Comment:
Through some initial digging, I don't think we can change the font size/color. (I might be wrong though)
If this is the case, should we use a https://api.slack.com/reference/block-kit/blocks#context block? It looks less dominated in terms of styles.

Reply:
Good idea, this stands out much less.